packages/web-components/fast-components/src/slider/fixtures/base.html

```diff
@@ -1,29 +1,30 @@
 <fast-design-system-provider use-defaults>
     <div style="display: flex; flex-direction: column; justify-content: space-evenly; width: 500px; height: 100vh">
-
-        <fast-slider orientation="vertical" min="0" max="100" step="10" value="70" style="height: 300px; width: 50px;">
-            <fast-slider-label
-                position="0"
-            >
-                0&#8451;
-            </fast-slider-label>            
-            <fast-slider-label
-                position="10"
-            >
-                10&#8451;
-            </fast-slider-label>
-            <fast-slider-label
-                position="90"
+        <div style="margin-bottom: 60px;">
+            <button style="width: 90px;margin: 10px;" onclick="const slider = document.getElementById('switcher'); slider.getAttribute('orientation') === 'horizontal' ? slider.setAttribute('orientation', 'vertical') : slider.setAttribute('orientation', 'horizontal');">Change orientation</button>
```

Comment:
That's some serious inline JS there. Any better way to handle this? I know storybook has some limitations, but maybe some code can just run a selector at startup and add a normal even handler?

Reply:
I tried a few simpler things I thought should work like an inline style block but couldn't seem to get it to resolve my function. I'll try again though. believe me I didn't want to do it that way :)

Reply:
I think I might need to become more of a storybook pro to get this looking nicer, I will do that but might take some time, do we want to hold this PR up or just remove this code or leave it as is until I get storybook fu upgraded?

Reply:
We can move ahead. It's not a huge issue.

Reply:
Ok, I'm pretty sure there are nice ways to test this in storybook.